qa/tasks/cephfs/test_volumes.py

```diff
@@ -6642,6 +6643,14 @@ def test_clone_subvolume_info(self):
         if subvol_info["type"] != "clone":
             raise RuntimeError("type should be set to clone")
 
+        for i in ['volume', 'subvolume', 'snapshot']:
```

Comment:
nit: for key in ...

Reply:
"key" is usually used in context of key-value pairs in dictionaries and never for list. So this is best left as it is.

Reply:
Typically we use "i" with `for` loops. It stands for items, items in list.

Reply:
Typically most docs use "i", so I'll keep with the standard convention.